src/main/java/net/runelite/rsb/wrappers/client_wrapper/BaseClientWrapper.java

```diff
@@ -885,6 +893,9 @@ public void setStringStackSize(int stackSize) {
         wrappedClient.setStringStackSize(stackSize);
     }
 
+    @Override
+    public void setCameraPitchTarget(int cameraPitchTarget) { wrappedClient.setCameraPitchTarget(cameraPitchTarget); }
```

Comment:
The spacing on this is inconsistent with the rest

Reply:
my bad, just realized intellij auto collapses braces, i just copied what I saw; will be fixed in next commit